src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -200,6 +203,16 @@ private static List<String> getTablesFromEntitySet(Set<? extends Entity> entitie
     return tableNames;
   }
 
+  private static List<String> getDatabasesFromEntitySet(Set<? extends Entity> entities) {
+    List<String> databaseNames = new ArrayList<>();
```

Comment:
I only noticed it now, but we probably should use Set here to remove duplicates (or maybe even better to return set instead of list).

The same issue exist for `getTablesFromEntitySet `

Reply:
done, tables will be fixed with partitions pr